internal/network/command.go

```diff
@@ -59,6 +62,13 @@ func New(prerunner pcmd.PreRunner) *cobra.Command {
 func printTable(cmd *cobra.Command, network networkingv1.NetworkingV1Network) error {
 	table := output.NewTable(cmd)
 
+	if network.Spec == nil {
+		return fmt.Errorf(errors.CorruptedNetworkResponseErrorMsg, "spec")
+	}
+	if network.Status == nil {
+		return fmt.Errorf(errors.CorruptedNetworkResponseErrorMsg, "status")
+	}
```

Comment:
Typically we would just print the empty string in place of these fields if they (for some reason) aren't returned by the backend.

Reply:
If the backend were to stop returning a single field (maybe it was renamed by accident), we still want the user to see as much of the output as possible.

Reply:
@ejagade Have we had odd responses that did not have `Spec` or `Status`?

@brianstrauch I think the idea is that if these checks were not in place, we could be returning a nil pointer dereference error to the user instead of a more meaning message if `Spec` or `Status` is missing since we are accessing their attributes below?🤔

Reply:
That's fair. At some point in the future, we'd like to be able to do `network.GetSpec().GetDisplayName()` but that currently a limitation of our SDK generator.

Reply:
> Have we had odd responses that did not have Spec or Status

Not that I know of. the intention is to have a safeguard in the code rather than panic.